src/functions/postResult/application/jwt-verification-service.ts

```diff
@@ -0,0 +1,74 @@
+import { StandardCarTestCATBSchema } from '@dvsa/mes-test-schema/categories/B';
+import * as logger from '../../../common/application/utils/logger';
+import jwtDecode = require('jwt-decode');
+
+export const verifyRequest = (headers: { [key: string]: string }, test: StandardCarTestCATBSchema): boolean => {
+
+  const employeeId = getEmployeeIdFromToken(headers.Authorization);
+  if (employeeId === null) {
+    logger.warn(`No valid authorisation token in request ${employeeId}`);
+    return null;
```

Comment:
Signature says that you're returning `boolean` (which makes sense, you're doing verification after all), but here you return `null`.

Reply:
have changed to only return true/false